cmd/wallet-web/src/components/InputField/InputField.vue

```diff
@@ -1,23 +1,22 @@
-<!--
- * Copyright SecureKey Technologies Inc. All Rights Reserved.
- *
- * SPDX-License-Identifier: Apache-2.0
--->
+/* Copyright SecureKey Technologies Inc. All Rights Reserved. SPDX-License-Identifier: Apache-2.0 */
```

Comment:
This is not an expected change, is it?

Reply:
dont know how this got edited probably by prettier i fixed though